src/AlgebraicGeometry/Schemes/AffineSchemes/Morphisms/Attributes.jl

```diff
@@ -164,6 +165,7 @@ function preimage(
 end
 
 function preimage(f::AbsSpecMor, Z::AbsSpec{<:Ring, <:MPolyRing}; check::Bool=true)
+  @check true
```

Comment:
Is `@check true` any different from `@check`?

This is one of several methods which have unused `check` arguments. Why is that so? Perhaps this is meant to enable potential future extensions of this methods which use `check` after all?

Reply:
> Is `@check true` any different from `@check`?

No. It is just a deprecated way in which I was writing it at first. 

I can't say anymore why there are so many methods which apparently don't use `check`. But sometimes it's also just easier to add that optional argument to all methods of a given function, even though only some use it. For instance: The `hom(A, B, img_gens)` methods to create maps of rings has a `check` argument. We want to write generic code which accepts all combinations of types `MPolyRing`, `MPolyQuoRing`, `MPolyLocRing`, `MPolyQuoLocRing` for `A` and `B`. Whenever there is a modulus involved in the domain `A`, there is some well-definedness-check which is eventually to be skipped. So we write the code as `hom(A, B, img_gens, check=false)` for internal calls with sane input. But then also the method with signature `A::MPolyRing` needs to have a dummy optional argument `check::Bool`, even though there's nothing to do there. 

Does that make sense?

Reply:
Yeah, that makes sense. Thanks